Userland/Utilities/unzip.cpp

```diff
@@ -162,6 +178,20 @@ int main(int argc, char** argv)
     args_parser.add_positional_argument(path, "File to unzip", "path", Core::ArgsParser::Required::Yes);
     args_parser.parse(argc, argv);
 
+    auto cwd = realpath(".", nullptr);
+    if (unveil(cwd, "rwc") < 0) {
+        fprintf(stderr, "CWD unveil\n");
+        perror("unveil");
```

Comment:
Using both `fprintf` and `perror` seems redundant. Simply `perror("unveil")` is sufficient.